nvram.h

```diff
@@ -1,79 +1,118 @@
 #ifndef INCLUDE_NVRAM_H
 #define INCLUDE_NVRAM_H
 
+// We prefix every "exposed" function with `libinject_` as we don't want to clobber
+// anything by default.
+// Internal methods will be prefixed with `_libinject_`.
```

Comment:
Maybe we should just make internal methods `static` so they can't affect other libraries and not worry about giving them a prefixed name?